#// Структура управления движением
  	uint32 startStop        # Стоим или двигаемся
	float32 radius          # Радиус по которому нужно двигаться
	float32 speed           # Скорость с которой нужно двигаться`
	uint32 command1         # Команда дполонительная
	uint32 command2         # Команда дполонительная